docs/03_tutorials/20_tic-tac-toe-game-smart-contract-single-node.md

```diff
@@ -111,15 +108,19 @@ info  2020-08-10T07:57:06.400 thread-0  producer_plugin.cpp:2134      produce_bl
 info 
 ```
 
+## Create Accounts and Key Pairs
+The game requires at least two blockchain accounts, one for each player. The tutorial also creates a blockchain account to load the smart contract. 
+
 ### Procedure for Accounts
 1. Create a wallet - [How To Create A Wallet](https://developers.eos.io/manuals/eos/v2.0/cleos/how-to-guides/how-to-create-a-wallet)
 2. Create key pair - [How To Create Key Pairs](https://developers.eos.io/manuals/eos/v2.0/cleos/how-to-guides/how-to-create-key-pairs)
 3. Create the account - [How To Create An Account](https://developers.eos.io/manuals/eos/v2.0/cleos/how-to-guides/how-to-create-an-account#gatsby-focus-wrapper)
 4. Import the account private key to the wallet - [How To Import A Key](https://developers.eos.io/manuals/eos/v2.0/cleos/how-to-guides/how-to-import-a-key)
```

Comment:
Suggestion: These three steps are fine as is, but it would be more clear and less prone to human error if they're performed below on a per account basis, especially when copy/pasting the generated keys. Something like this:

For **host** account:
1. create key pair
2. create the account
3. import private key

For **challenger** account:
1. create key pair
2. create the account
3. import private key

For **tictactoe** account:
1. create key pair
2. create the account
3. import private key

Moreover, if I were to host the tictactoe game and play it with a friend I would ask him to follow the instructions as a challenger. He wouldn't need to create three key pairs, etc. Therefore, organizing the steps per account makes more sense.

Reply:
hi, feel free to raise a jira ticket for this one.